src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/PasswordController.kt

```diff
@@ -37,6 +36,8 @@ class PasswordController(
     fun setPasswordForUser(username: String): String
     {
         val password = context.postData<SetPassword>().password
+        logger.debug(password)
+        logger.debug(username)
```

Comment:
Debug statements need removing - particularly since it has a password in it!

Reply:
Oops, good spot!